doc/Dataplane Code Gen/DataplaneCodeGen_Quickstart_Tutorial.md

```diff
@@ -0,0 +1,185 @@
+
+**Learn more**: see other features listed in the [Azure.Template.DataplaneCodeGen README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-LLC/Azure.Template.LLC/README.md).
+
+### Create the project
+
+#### 1. Copy the Azure.Template.DataplaneCodeGen directory
+
+Copy the `sdk\template-dataplanecodegen` directory to an `sdk\<your-service-name>` directory at the same level. For example, from the root of your cloned repo:
+
+```
+% mkdir sdk\<your-service-name>
+% xcopy sdk\template-dataplanecodegen sdk\<your-service-name> /E
+```
+
+**Learn more:** to understand more about the Azure SDK repo structure, see [Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md) in the `azure-sdk` repo.
+
+#### 2. Remove existing generated code
+
+Open the `Azure.Template.DataplaneCodeGen.sln` file from the `sdk\<your-service-name>\Azure.Template.DataplaneCodeGen` directory.
+
+You'll notice that the `Azure.Template.DataplaneCodeGen` project already has generated code in it!  You're welcome to explore what's there -- it's been generated from the swagger file in the `swagger` folder. For this tutorial, however, we'll delete these generated files so we can start clean.
+
+- Delete the Generated folder in the `Azure.Template.DataplaneCodeGen` project.
+- Delete the `swagger` folder in the `Azure.Template.DataplaneCodeGen` project.
+
+#### 3. Rename the solution and projects
+
+In Visual Studio:
+
+- Rename the `Azure.Template.DataplaneCodeGen` project to `Azure.<your-sdk-name>`.
+- Rename the `Azure.Template.DataplaneCodeGen` solution to `Azure.<your-sdk-name>`.
+- Remove the `Azure.Template.DataplaneCodeGen.Tests` project from the solution.
+
+**Note**: `Azure.<your-sdk-name>` is the Azure SDK package name.
+
+**Learn more:** see the [.NET Guidelines on Namespace Naming](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespace-naming) for more information on naming Azure SDK packages. This name diverges from approved Azure SDK namespaces, since it is intended to be used just for this tutorial.
+
+#### 4. Save and commit your changes to your branch
+
+This will make it easier to see the changes you make when you generate code in the clean solution.
+
+```
+% git add -A
+% git commit -m "initial commit for <service-name>"
+```
+
+### Create the generated layer
+
+In this section, we'll create a generated API layer built on Azure Core. We'll use a small swagger file containing a REST API definition for Azure Messaging WebPubSub.
+
+**Learn more:** the [autorest.csharp README](https://github.com/Azure/autorest.csharp#setup) has great samples showing how to customize the API in the generated code. Explore this further as you design APIs for your own service.
```

Comment:
Instead of saying "customize" here, what about saying "add convenience APIs"?  Does that sound correct?